meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb

```diff
@@ -1,5 +1,5 @@
 SRC_URI = "git://github.com/GENIVI/hmi-layout-gdp.git"
-SRCREV = "d0ca9d072012d8ac333537d9b922d9242e022efe"
+SRCREV = "79a4f69e7588a6efc4b4c95abec7794216588087"
```

Comment:
Ah I see now that you changed the name in HMI itself. In this case, the update of HMI here is the real change and renaming is just a side-effect, from the point of view of meta-genivi-dev. Hence I'd like the commit log to be changed.